src/topology/algebra/infinite_sum.lean

```diff
@@ -689,6 +703,9 @@ variables {f g : β → α} {a a₁ a₂ : α}
 lemma has_sum_le (h : ∀b, f b ≤ g b) (hf : has_sum f a₁) (hg : has_sum g a₂) : a₁ ≤ a₂ :=
 le_of_tendsto_of_tendsto' hf hg $ assume s, sum_le_sum $ assume b _, h b
 
+@[mono] lemma has_sum_mono (hf : has_sum f a₁) (hg : has_sum g a₂) (h : f ≤ g) : a₁ ≤ a₂ :=
```

Comment:
Can you explain what this brings compared to `has_sum_le`? Same thing for `tsum_mono` below.

Reply:
Consistency with `finset.sum_mono`, primarily. Also, `tsum_strict_mono` is elegible for tagging with `@[mono]`, while `tsum_lt_tusm` is not.

Reply:
Then I'd say remove `has_sum_le`. We don't need two statements saying exactly the same thing modulo a permutation of the arguments, unless there is a very good reason to do so. And maybe rename this one to `has_sum.mono`, for use with dot notation as @bryangingechen suggests?

Reply:
This isn't just an argument permutation, one is stated as `∀b, f b ≤ g b` while the other is `f ≤ g`

Reply:
This is defeq, right?

Reply:
As for dot notation, `hf.mono hg h` reads a little oddly to me as it doesn't really have the same pattern as `hf.add hg`, but I'll let you decide.

Reply:
Yes, it's defeq for `≤`, but not defeq for `<`. I think we should keep the API for `≤` and `<` the same as each other for discoverability - but I wouldn't object to dropping the quantified versions of the lemmas entirely.